static/js/components/applications/detail_sections.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import * as R from "ramda"
+
+import ProgressDetailRow from "./ProgressDetailRow"
+import { formatReadableDateFromStr } from "../../util/util"
+import { PAYMENT, PROFILE_VIEW, REVIEW_STATUS_REJECTED } from "../../constants"
+
+import type { DrawerChangePayload } from "../../reducers/drawer"
+import type {
+  ApplicationDetail,
+  ApplicationRunStep,
+  ApplicationSubmission
+} from "../../flow/applicationTypes"
+import type { User } from "../../flow/authTypes"
+
+type DetailSectionProps = {
+  ready: boolean,
+  fulfilled: boolean,
+  openDrawer: (actionPayload: DrawerChangePayload) => void
+}
+
+type ProfileDetailProps = DetailSectionProps & {
+  user: User
+}
+
+export const ProfileDetail = (props: ProfileDetailProps): React$Element<*> => {
```

Comment:
I think each of these components should be in their own source files for consistency

Reply:
They're all very small components, they're all very closely related, and it's somewhat likely that a change to one would need to be applied to all. I know the normal practice is 1 component 1 file, but given what I mentioned + the fact that this would result in 12 separate files when you include test files, I'd prefer to keep it as-is if you don't feel that strongly about it.